app/models/patient.rb

```diff
@@ -26,7 +26,7 @@ class Patient
 
   before_validation :clean_fields
   before_save :save_identifier
-
+  before_update :update_sent_by_sent_at
```

Comment:
I think this name is close but doesn't quite capture the important thing, which is the pledge! Maybe `update_pledge_sent_by_sent_at`?

Reply:
Sounds good, will update it